app/modules/ads/setup.js

```diff
@@ -147,6 +147,14 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
     context.set('custom.isCMPEnabled', true);
   }
 
+  const btlConfig = instantGlobals.wgAdDriverBillTheLizardConfig;
+
+  if (context.get('options.slotRepeater') && btlConfig && btlConfig.projects && btlConfig.projects.cheshirecat
```

Comment:
Consider extracting the conditions to a separate function, so it's readable. I wonder if it's possible that a model won't have countries list.

Reply:
Refactored. It's not possible: it's a required field.